lib/DDG/Spice/Snow.pm

```diff
@@ -17,7 +17,7 @@ attribution web => [ 'https://www.duckduckgo.com', 'DuckDuckGo' ],
             github => [ 'https://github.com/duckduckgo', 'DuckDuckGo'],
             twitter => ['http://twitter.com/duckduckgo', 'DuckDuckGo'];
 
-spice to => 'http://isitsnowingyet.org/api/check/$1/key/{{ENV{DDG_SPICE_SNOW_APIKEY}}}';
+spice to => 'https://duckduckgo.com/js/spice/snow/$1';
```

Comment:
This isn't necessary.

The `spice to` url is where the actual request is made to. The `duckduckgo.com/js/spice/snow/$1` call is re-written via NGXINX to the `spice to` url. This way the client makes a request to DDG and we in turn make a request to the API to maintain your privacy.

Please switch this back, otherwise the IA will break in production.